pkg/cmd/listen.go

```diff
@@ -164,6 +166,15 @@ func (lc *listenCmd) runListenCmd(cmd *cobra.Command, args []string) error {
 		NoWSS:               lc.noWSS,
 	}, lc.events)
 
+	if lc.onlyPrintSecret {
+		secret, err := p.GetSessionSecret(context.Background())
+		if err != nil {
+			return err
+		}
+		fmt.Printf("%s", secret)
```

Comment:
```suggestion
		fmt.Printf("%s\n", secret)
```
Add a newline so that command-line output looks cleaner?